getting-started/dc2-guide.rst

```diff
@@ -0,0 +1,190 @@
+
+This command should print a list of data IDs that meet the search criteria, along with their tract, patch, and visit number.
+Certain arguments are required after the ``--where``, including ``skymap`` and ``instrument``, while most others are optional.
+
+Processing Data
+===============
+
+Now it's time to process some data.
+In this guide, we will start with raws, run "processCcd" (which includes :py:mod:`lsst.ip.isr.IsrTask`, :py:mod:`lsst.pipe.tasks.characterizeImage.CharacterizeImageTask`, and :py:mod:`lsst.pipe.tasks.calibrate.CalibrateTask`), and make good seeing coadd templates.
+
+In a second pipeline, we will run difference imaging using the templates we just built and save the results in an Alert Production Database (APDB).
+
+Building good seeing templates
+------------------------------
+
+Here is the pipeline we will use.
+Note that multiple python configuration options can be used by typing a placeholder continuation character (e.g., ``|``) followed by one python config declaration per line.
+
+.. prompt:: yaml
+
+   description: An AP pipeline for building templates with LsstCam-imSim data
+
+   instrument: lsst.obs.lsst.LsstCamImSim
+   imports:
+   - location: $AP_PIPE_DIR/pipelines/ApTemplate.yaml
+   tasks:
+     calibrate:
+       class: lsst.pipe.tasks.calibrate.CalibrateTask
+       config:
+         connections.astromRefCat: 'cal_ref_cat_2_2'
+         connections.photoRefCat: 'cal_ref_cat_2_2'
+         astromRefObjLoader.ref_dataset_name: 'cal_ref_cat_2_2'
+         photoRefObjLoader.ref_dataset_name: 'cal_ref_cat_2_2'
+         python: |
+         config.astromRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
+         config.photoRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
+   subsets:
+     singleFrameAp:
+     subset:
+       - isr
+       - characterizeImage
+       - calibrate
+       - consolidateVisitSummary
+     description: >
+       Tasks to run for single frame processing that are necessary to use the good seeing selector to build coadds for use as difference imaging templates.
+
+This example pipeline imports a pipeline from :py:mod:`lsst.ap.pipe` you may `view on GitHub <https://github.com/lsst/ap_pipe/blob/master/pipelines/ApTemplate.yaml>`__.
+There are some special configurations concerning reference catalogs that must be set for this camera and/or dataset, so the example pipeline above lists the ``calibrate`` task explicitly to add custom configurations.
+
+To run this example pipeline, save it as ``ApTemplate-DC2.yaml``, choose an appropriate output collection name (``u/USERNAME/OUTPUT-COLLECTION-1`` in the example below), and run
+
+.. prompt:: bash
+
+   pipetask run -j 12 -b /repo/dc2 -d "band='g' AND skymap='DC2' AND tract=3829" -i 2.2i/defaults/test-med-1 -o u/USERNAME/OUTPUT-COLLECTION-1 -p ApTemplate-DC2.yaml#singleFrameAp --register-dataset-types
```

Comment:
For what it's worth, this isn't working for me; I get a bunch of errors around brighter fatter kernel correction - the same error reported in these places:
 - https://lsstc.slack.com/archives/C4JQP6FRS/p1623342592112800
 - https://lsstc.slack.com/archives/C01FBUGM2CV/p1625579222311800
 - https://lsstc.slack.com/archives/C025P0D9SKB/p1624470090098600

---

I _think_ things work with just the magic `connections.newBFKernel: bfk` parameter. That is, I added this stanza under `tasks:`:
```yaml
isr:
  class: lsst.ip.isr.IsrTask
  config:
    connections.newBFKernel: bfk
    doBrighterFatter: True
```

It hasn't yet finished, but it has at least gotten further, so I think it's working.